.gitignore

```diff
@@ -1,6 +1,42 @@
+# Temp files
+*~
+
+# Databases
+dcrd.db
+*-shm
+*-wal
+
+# Log files
+*.log
+
+# Compiled Object files, Static and Dynamic libs (Shared Objects)
+*.o
+*.a
+*.so
+
+# Folders
+_obj
+_test
+vendor
+
+# Architecture specific extensions/prefixes
+*.[568vq]
+[568vq].out
+
+*.cgo1.go
+*.cgo2.c
+_cgo_defun.c
+_cgo_gotypes.go
+_cgo_export.*
+
+_testmain.go
+
+# Binaries
+*.exe
 dcrd
+cmd/addblock/addblock
+cmd/checkdevpremine/checkdevpremine
 cmd/dcrctl/dcrctl
-cmd/promptsecret/promptsecret
-*~
-*.pyc
```

Comment:
did you mean to remove this?

Reply:
Yes.  There is no longer any Python code in the project.